packages/idx/src/map.component.ts

```diff
@@ -7,6 +7,7 @@
 import _ from 'lodash'
 import {Stratus} from '@stratusjs/runtime/stratus'
 import * as angular from 'angular'
+// @ts-ignore
```

Comment:
this shouldn't be an issue once you `yarn install` as there is a `@types/numeral` package to validate this

Reply:
I was able to remove it.